apps/api/v1/test/lib/selected-calendars/_post.test.ts

```diff
@@ -79,7 +79,7 @@ describe("POST /api/selected-calendars", () => {
         integration: "google",
         externalId: "ext123",
         userId: 444444,
-        id: "xxx-xxx",
+        id: "f47ac10b-58cc-4372-a567-0e02b2c3d479",
```

Comment:
Zod validation included `.uuid()` so we need to pass an actual uuid for this to pass.